<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>371</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string>DAQInterface</string>
    </property>
    <widget class="QPlainTextEdit" name="taDAQInterface">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>351</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="bDAQInterface">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>380</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Start DAQInterface</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Messages</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bEndSession">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>380</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>End Session</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>62</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>status:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbStatus">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>340</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>null</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bBelen">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>340</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Para Belen</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>671</width>
      <height>491</height>
     </rect>
    </property>
    <property name="title">
     <string>DAQInterface Commands</string>
    </property>
    <widget class="QPushButton" name="bDAQcomp">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>160</y>
       <width>161</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Components</string>
     </property>
    </widget>
    <widget class="QListView" name="lvComponents">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>141</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
    <widget class="QListView" name="lvConfigurations">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>150</y>
       <width>141</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Components</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Configurations</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bDAQconf">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>190</y>
       <width>161</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Configurations</string>
     </property>
    </widget>
    <widget class="QListView" name="lvConfigBOOT">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>150</y>
       <width>141</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>120</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>BOOT Configurations</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bBOOT">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>220</y>
       <width>161</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>BOOT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bCONFIG">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>250</y>
       <width>161</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>CONFIGURATE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bBOOTandCONFIG">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>280</y>
       <width>161</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>BOOT AND CONFIG</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>320</y>
       <width>181</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>Transition Commands</string>
     </property>
     <widget class="QPushButton" name="bStart">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>START</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bStop">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>STOP</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bPause">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>PAUSE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bTerminate">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>TERMINATE</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Experiment:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboExperiment">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>211</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Profiles:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboProfiles">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>211</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="bNewExperiment">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>51</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>New</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bNewProfile">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>70</y>
       <width>51</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>New</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bEditExperiment">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>30</y>
       <width>51</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bDeleteExperiment">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>30</y>
       <width>51</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bEditProfile">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>70</y>
       <width>51</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bDeleteProfile">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>70</y>
       <width>51</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="bDebug">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Debug</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSource_config_file">
   <property name="text">
    <string>Source config file</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
